### PR TITLE
✅ BoardConverterTestCase.test_convert_to_string()

### PR DESCRIPTION
**Pull request by alena-bartosh**
```
python -m unittest ./tests/test_BoardConverter.py
python -m unittest ./tests/test_Board.py
```

### CODE REVIEW

File: tests/test_BoardConverter.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+class BoardConverterTestCase(unittest.TestCase):
+    def test_convert_to_string(self):
+        board = Board(3, 4)
+        points = [Point(1, 2), Point(1, 3), Point(2, 0), Point(2, 1), Point(2, 2), Point(2, 3)]
+        board.set_cells_with_value(points, True)
+        # ----
+        # --xx
+        # xxxx
+
+        expected = '0 0 0 0\n0 0 1 1\n1 1 1 1\n'
```
Comment (s373r) [inline tests/test_BoardConverter.py]: please add line breaks into value

**Review by s373r — CHANGES_REQUESTED** [review]

File: tests/test_BoardConverter.py (hunk 1 above)
Comment (alena-bartosh) [reply tests/test_BoardConverter.py]: Good point

**Review by s373r — DISMISSED** [review]

File: tests/test_BoardConverter.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+        self.assertEqual(expected, BoardConverter.to_string(board))
+
+    def test_convert_from_string_when_correct_cells_line(self):
+        # TODO(AB): Task for Serg
```
Comment (Serg-Bartosh) [inline tests/test_BoardConverter.py]: please change serg --> Onehayzent ,@alena-bartosh

Comment (alena-bartosh) [reply tests/test_BoardConverter.py]: Done, dear Onehayzent

File: tests/test_BoardConverter.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+from board import Board
+from board_converter import BoardConverter
+from point import Point
+
+
+class BoardConverterTestCase(unittest.TestCase):
+    def test_convert_to_string(self):
+        board = Board(3, 4)
+        points = [Point(1, 2), Point(1, 3), Point(2, 0), Point(2, 1), Point(2, 2), Point(2, 3)]
```
Comment (Serg-Bartosh) [inline tests/test_BoardConverter.py:11]: perhaps it is worth making a row from `Point`?

point : .. / 
point: ../
...

Comment (alena-bartosh) [reply tests/test_BoardConverter.py:11]: Tnx for proposition

**Review by s373r — APPROVED** [review]

**Review by Serg-Bartosh — APPROVED** [review]
this is what a hard worker looks like :smile:

File: tests/test_BoardConverter.py (hunk 2 above)
Comment (Serg-Bartosh) [reply tests/test_BoardConverter.py]: thx

**Pull request merged**